internal/slicer/slicer_test.go

```diff
@@ -755,8 +779,91 @@ var slicerTests = []slicerTest{{
 	},
 	error: `slice test-package_myslice: content is not a file: /x/y`,
 }, {
-	summary: "Non-default archive",
+	summary: "Multiple archives with priority",
```

Comment:
Can we change to "Infer default archive from multiple archives with priorities"?

Reply:
That's not exactly the purpose of this test actually. In this test, `default` field is obsolete anyway. The test only checks the addition of `priority` fields of multiple archives.